qpc/cli.py

```diff
@@ -218,10 +218,10 @@ def main(self):
                 logger.error(_(messages.SERVER_CONFIG_REQUIRED), QPC_VAR_PROGRAM_NAME)
                 sys.exit(1)
 
-        if read_require_auth():
-            if (not is_server_cmd or is_server_logout) and not read_client_token():
-                logger.error(_(messages.SERVER_LOGIN_REQUIRED), QPC_VAR_PROGRAM_NAME)
-                sys.exit(1)
+            if read_require_auth():
+                if (not is_server_cmd or is_server_logout) and not read_client_token():
```

Comment:
So if I already did `qpc server config` and try `qpc server logout`, it will tell me that I need to log in first :upside_down_face: 

I guess it's not wrong, but if we think about commands as ways to achieve a certain state, then that state is already achieved and there's no reason to warn the user.